sdk/storagemover/Azure.ResourceManager.StorageMover/CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Release History
```

Comment:
It seems that the CHANGELOG and README files are out of date and don't follow our requirement, could you please take a look at this [PR](https://github.com/Azure/azure-sdk-for-net/pull/32935) and update the files correspondingly?

Reply:
Done